sonic_platform_base/sonic_xcvr/utils/__init__.py

```diff
@@ -0,0 +1 @@
+# utils package for transceiver common utilities
```

Comment:
no need?

Reply:
I added one new cache.py, so added this file, will try confirm it.